functions/src/repository/info.ts

```diff
@@ -11,17 +12,19 @@ export interface RepositoryInfoInput {
 }
 
 export const getRepositoryInfo: any = async (token: string, fullName: string) => {
-  const data: [GitHubRepositoryInput, GitHubPullRequestInput[], GitHubEventInput[], GitHubReleaseInput[]] = await Promise.all([
+  const data: [GitHubRepositoryInput, GitHubPullRequestInput[], GitHubEventInput[], GitHubReleaseInput[], GitHubIssuesInput[]] = await Promise.all([
     GitHubClient<GitHubRepositoryInput>(`/repos/${fullName}`, token),
     GitHubClient<GitHubPullRequestInput[]>(`/repos/${fullName}/pulls?state=open`, token),
     GitHubClient<GitHubEventInput[]>(`/repos/${fullName}/events`, token),
     GitHubClient<GitHubReleaseInput[]>(`/repos/${fullName}/releases`, token),
+    GitHubClient<GitHubIssuesInput[]>(`/repos/${fullName}/issues`, token),
   ]);
   const mappedData: GitHubRepositoryModel = {
     ...GitHubRepositoryMapper.import(data[0], 'all'),
     pullRequests: data[1].map((pullrequest: GitHubPullRequestInput) => GitHubPullRequestMapper.import(pullrequest)),
     events: data[2].map((event: GitHubEventInput) => GitHubEventMapper.import(event)),
     releases: data[3].map((release: GitHubReleaseInput) => GitHubReleaseMapper.import(release)),
+    issues: data[4].map((issues: GitHubIssuesInput) => GitHubIssuesMapper.import(issues)),
```

Comment:
I think, this line should be...

```typescript
issues: data[4].map((issue: GitHubIssuesInput) => GitHubIssuesMapper.import(issue)),
```

*Note: The `Input` and `Mapper` should be named singular but this won't effect the functionality*